github/ci/prow-deploy/tasks/deploy.yaml

```diff
@@ -1,3 +1,32 @@
+# FIXME: use an image with this binary installed
+- name: install kustomize
+  unarchive:
+    src: https://github.com/kubernetes-sigs/kustomize/releases/download/kustomize/v3.8.7/kustomize_v3.8.7_linux_amd64.tar.gz
+    dest: /usr/local/bin
+    remote_src: yes
+    mode: a+x
+
+# FIXME: use an image with this binary installed
+- name: install bazelisk
+  get_url:
+    url: https://github.com/bazelbuild/bazelisk/releases/download/v1.7.4/bazelisk-linux-amd64
+    dest: /usr/local/bin/bazelisk
```

Comment:
I /usr/local/bin usable as normal user ? I think you need to be at least in a privileged group

Reply:
Sure, accessible to normal users, most of the system-wide binaries live there.

Reply:
Sorry, I meant writable.

Reply:
Good catch, as I see it it would depend on the usage we want to give to the role, if we want it to be something self-contained and usable by anyone to deploy prow then it should install all the required dependencies without making assumptions about if the user applying the role is root or not; if we want the role to be something used on CI systems then it can rely on some dependencies already installed on the hosts, or assume that the role will be applied by root. 

Going for the first more generic approach for now, we can change later, now the binaries are installed under /tmp, PTAL